src/app/(admin)/admin/blog/page.tsx

```diff
@@ -0,0 +1,5 @@
+const bloghome = () => {
+  return <div> bloghome</div>;
```

Comment:
have you run npm install for the linter and also installed prettier / set up format on save? if not could you do that pls :)

Reply:
i have done that :/

Reply:
ok cool, nothing to do with this code lol just popped into my head